src/SideBar/InboxButton/InboxButton.js

```diff
@@ -0,0 +1,11 @@
+const InboxButton = (props) => {
+    const unreadEmailCount = props.allEmailSnippets.filter((obj) => obj.read === 1).length;
+    return (
+        <div>
+            <button id="inboxButtons" type="button" className="btn btn-info w-100 text-left d-flex flex-row justify-content-between">Inbox
```

Comment:
this class should be called `inboxButton` without the s

Reply:
done